csharp/ql/test/library-tests/types/types.cs

```diff
@@ -56,4 +56,15 @@ interface GenericInterface<T> { }
     struct GenericStruct<T> { }
     class Map<U, V> { }
 
+    [System.Runtime.CompilerServices.InlineArray(10)]
+    public struct MyInlineArray
+    {
+        private int myInlineArrayElements;
+    }
+
+    [System.Runtime.CompilerServices.InlineArray(5)]
+    public struct MyMultiDimensionalInlineArray
+    {
+        private MyInlineArray myMultiDimentionalInlineArrayElements;
```

Comment:
Could you add test cases with `int[][]` and `int[,]` as inline array field types too? That would test the third case of `getDimension()`.

Reply:
Sure! I will add some more test coverage! 😄